manifests/z/Zoho/ZohoMail/Desktop/1.3.2/Zoho.ZohoMail.Desktop.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Zoho.ZohoMail.Desktop
+PackageVersion: 1.3.2
+PackageLocale: en-US
+Publisher: Zoho Corporation

```

Comment:
```suggestion
Publisher: Zoho Mail
```

Reply:
i think publisher should be Zoho corporation as in they are the publisher of said app